code/src/pages/dashboard-order-details/DashboardOrderDetailsPage.tsx

```diff
@@ -0,0 +1,49 @@
+import { useParams } from "react-router-dom";
+
+import OrderItem from "@/containers/order-item/OrderItem";
+
+import AppLoader from "@/components/app-loader/AppLoader";
+
+import { useLocaleContext } from "@/context/i18n/I18nProvider";
+import useErrorPageRedirect from "@/hooks/use-error-page-redirect/useErrorPageRedirect";
+import { dashboardOrderDetailsPageErrorConfig } from "@/pages/dashboard-order-details/DashboardOrderDetails.constants";
+import { useGetAdminOrderByIdQuery } from "@/store/api/ordersApi";
+import { OrderId } from "@/types/order.types";
+
+import "@/pages/dashboard-order-details/DashboardOrderDetailsPage.scss";
+
+type DashboardOrderDetailsPageParams = {
+  orderId: OrderId;
+};
+
+const DashboardOrderDetailsPage = () => {
+  const { orderId } = useParams<DashboardOrderDetailsPageParams>();
+  const { locale: lang } = useLocaleContext();
+  const { renderRedirectComponent } = useErrorPageRedirect();
+
+  if (!orderId) {
+    return renderRedirectComponent(dashboardOrderDetailsPageErrorConfig);
+  }
+
+  const { data: orderData, isLoading } = useGetAdminOrderByIdQuery({
+    orderId,
+    lang
+  });
```

Comment:
also handle `error` from the hook

Reply:
error is handled by global error handler 
![image](https://github.com/user-attachments/assets/6e0f8aec-eb15-4c64-b7a0-a28749878732)

Reply:
> error is handled by global error handler ![image](https://private-user-images.githubusercontent.com/118522791/352969616-6e0f8aec-eb15-4c64-b7a0-a28749878732.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyNDE2MjUsIm5iZiI6MTcyMjI0MTMyNSwicGF0aCI6Ii8xMTg1MjI3OTEvMzUyOTY5NjE2LTZlMGY4YWVjLWViMTUtNGM2NC1iN2EwLWEyODc0OTg3ODczMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyOVQwODIyMDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMzI2OTVkZDUyMWY5MGY0NWM0ZWM5YWZiMGZhNDAyNTY5MjM0Zjc1ODVmZmQ2NDI0Yjk1NTA3NWJkMTFmNzc4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.936K4RWPMFcRBL2kog_t8qVugD-jxWDNI_BwTDaBu7E)

Ok. But `isSnackbarHidden` got not from response. We should to set and hide it like property `false` ?

Reply:
By default, for `GET` requests it's hidden. But if you set this `true` it will show the error from the response

Reply:
> By default, for `GET` requests it's hidden. But if you set this `false` it will show the error from the response

Sure as you mentioned it's hidden. Is `isSnackbarHidder` exist in your error response ? 
I guess you like reassign this default to `false` but not from response

Reply:
> By default, for `GET` requests it's hidden. But if you set this `false` it will show the error from the response

ok gotted.